edx_proctoring/api.py

```diff
@@ -535,7 +537,7 @@ def add_bulk_allowances(exam_ids, user_ids, allowance_type, value):
         if multiplier < 0:
             raise AllowanceValueNotAllowedException(err_msg)
 
-    if allowance_type == constants.ADDITIONAL_TIME:
+    if allowance_type == ProctoredExamStudentAllowance.ADDITIONAL_TIME_GRANTED[0]:
```

Comment:
Why is the `ProctoredExamStudentAllowance.ADDITIONAL_TIME_GRANTED` an array?

Reply:
In the model.py file ADDITIONAL_TIME_GRANTED is actually an array so I needed the key text for this and not the helper text for the variable.

Reply:
I see. Look at this function: https://github.com/edx/edx-proctoring/blob/master/edx_proctoring/models.py#L712. That's what you want to use instead of directly comparing with the model constant.